test/features/install_jobs_service.feature

```diff
@@ -53,6 +53,39 @@ Feature: Install Kogito Jobs Service
 
     Then HTTP GET request on service "jobs-service" with path "jobs/1" is successful within 1 minutes
 
+#####
+
+  @persistence
+  @postgresql
+  Scenario: Install Kogito Jobs Service with persistence using PostgreSQL
+    Given Kogito Operator is deployed
+    And PostgreSQL instance "postgresql" is deployed within 3 minutes with configuration:
+      | username | myuser |
+      | password | mypass |
+      | database | mydb   |
+
+    When Install Kogito Jobs Service with 1 replicas with configuration:
+      | config      | database-type                   | PostgreSQL                             |
+      | runtime-env | QUARKUS_DATASOURCE_JDBC_URL     | jdbc:postgresql://postgresql:5432/mydb |
+      | runtime-env | QUARKUS_DATASOURCE_REACTIVE_URL | postgresql://postgresql:5432/mydb      |
```

Comment:
is reactive url needed for apps ? 
I think it is only jdbc supported: https://github.com/kiegroup/kogito-apps/blob/master/jobs-service/jobs-service-postgresql/pom.xml#L26-L27

Reply:
It was needed. When I tried to use only JDBC then the Jobs service started, however it wasn't able to fetch jobs from database.